save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -70,28 +71,20 @@ internal class OrganizationController(
 ) {
     private val webClientToPreprocessor = WebClient.create(config.preprocessorUrl)
 
-    @GetMapping("/all")
+    @PostMapping("/all-by-filters")
     @PreAuthorize("permitAll()")
     @Operation(
-        method = "GET",
+        method = "POST",
         summary = "Get all organizations",
         description = "Get organizations",
     )
     @Parameters(
-        Parameter(
-            name = "onlyActive",
-            `in` = ParameterIn.QUERY,
-            description = "Whether deleted organizations should be excluded from the response. The default is false.",
-            required = false
-        ),
+        Parameter(name = "organizationFilters", `in` = ParameterIn.DEFAULT, description = "organization filters", required = true),
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched all registered organizations")
-    fun getAllOrganizations(
-        @RequestParam(required = false, defaultValue = "false") onlyActive: Boolean
-    ): Mono<OrganizationDtoList> = when {
-        onlyActive -> getFilteredOrganizationDtoList(OrganizationFilters.created).collectList()
-        else -> blockingToMono { organizationService.findAll().map(Organization::toDto) }
-    }
+    fun getAllOrganizationsByFilters(
+        @RequestBody(required = true) organizationFilters: OrganizationFilters
```

Comment:
btw, why we use `Filters` as plural? it's a single object. Maybe rename everything?
It can be done in next PR

Reply:
Deleted

Reply:
Thanks, I'm thinking of doing it in https://github.com/saveourtool/save-cloud/issues/1538